CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 - none yet
 
+## [v0.2.0]
+## Removed
+- ⚠️ `pouchdb` and `pouchdb-find`, see [offline documentation](https://github.com/cozy/cozy-client-js/blob/master/docs/offline.md)
```

Comment:
s/`pouchdb` and `pouchdb-find`/`pouchdb` and `pouchdb-find` required dependencies/